cypress/e2e/item/authorization/itemLogin/itemLoginSetting.cy.ts

```diff
@@ -0,0 +1,107 @@
+import {
+  ItemLoginSchemaType,
+  PackedFolderItemFactory,
+  PackedItem,
+  PermissionLevel,
+} from '@graasp/sdk';
+
+import { v4 } from 'uuid';
+
+import { buildItemPath } from '../../../../../src/config/paths';
+import {
+  ITEM_LOGIN_SCREEN_FORBIDDEN_ID,
+  SHARE_ITEM_PSEUDONYMIZED_SCHEMA_ID,
+  buildShareButtonId,
+} from '../../../../../src/config/selectors';
+import { MEMBERS } from '../../../../fixtures/members';
+import { ITEM_LOGIN_PAUSE } from '../../../../support/constants';
+
+const checkItemLoginSetting = ({
+  isEnabled,
+  mode,
+  disabled = false,
+}: {
+  isEnabled: boolean;
+  mode: string;
+  disabled?: boolean;
```

Comment:
What is the difference between `isEnabled = true` and `disabled = false`? To me, the two parameters represent the same thing 😅.